lib/IRGen/GenType.cpp

```diff
@@ -1380,8 +1380,9 @@ TypeConverter::TypeConverter(IRGenModule &IGM)
   }
 
   bool error = readLegacyTypeInfo(*fs, path);
-  if (error)
-    llvm::report_fatal_error("Cannot read '" + path + "'");
+// avoids error: "<unknown>:1:1: Cannot read '/usr/lib/swift/layouts-x86_64.yaml'"
```

Comment:
The proper way to work around these is to ensure that your resource directory is properly formed. If you have built a custom toolchain, you will need to verify that these layout back-compat files are present under `My-Custom-Toolchain.xctoolchain/usr/lib/swift`.

Reply:
Please also note that these paths are subject to VFS remapping, and are sensitive to the `-resource-dir` flag.